ibm/service/configurationaggregator/data_source_ibm_config_aggregator_configurations.go

```diff
@@ -27,65 +27,82 @@ func DataSourceIbmConfigAggregatorConfigurations() *schema.Resource {
 		ReadContext: dataSourceIbmConfigAggregatorConfigurationsRead,
 
 		Schema: map[string]*schema.Schema{
-			"config_type": &schema.Schema{
+			"config_type": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The type of resource configuration that are to be retrieved.",
 			},
-			"service_name": &schema.Schema{
+			"service_name": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The name of the IBM Cloud service for which resources are to be retrieved.",
 			},
-			"resource_group_id": &schema.Schema{
+			"resource_group_id": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The resource group id of the resources.",
 			},
-			"location": &schema.Schema{
+			"location": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The location or region in which the resources are created.",
 			},
-			"resource_crn": &schema.Schema{
+			"resource_crn": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The crn of the resource.",
 			},
-			"prev": &schema.Schema{
+			"sub_account": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Add Validation function to check the condition described in docs
```
The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `[a-zA-Z0-9]`
```

Reply:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/944c92e3-0531-459e-b62e-171eef5e78e4">
<img width="1265" alt="image" src="https://github.com/user-attachments/assets/a1c98821-9973-4c27-906d-afed8b5d23e4">
<img width="1112" alt="image" src="https://github.com/user-attachments/assets/d009536f-1d6d-4263-b021-44fcc6f27e01">
<img width="1150" alt="image" src="https://github.com/user-attachments/assets/34175660-1324-4967-b939-facd2f8c3e67">